pkg/distributor/validate.go

```diff
@@ -355,9 +353,9 @@ func (m *MetadataValidationMetrics) DeleteUserMetrics(userID string) {
 
 func NewMetadataValidationMetrics(r prometheus.Registerer) *MetadataValidationMetrics {
```

Comment:
`MetadataValidationMetrics` and similar don't need to be exported anymore.

Reply:
I have fixed that. Thank you.